benchmarks/pom.xml

```diff
@@ -132,7 +132,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
-                <version>2.2</version>
+                <version>3.2.1</version>
```

Comment:
Are we able to remove the version (is it set in third-party?)

Reply:
done